backend/src/database/User.js

```diff
@@ -13,11 +13,21 @@ const createNewUser = async (newUser) => {
     `insert into users(id, name, username, email, highest_score) values($1, $2, $3, $4, $5) returning *`,
     values,
   );
-
+  // console.log('Result:', res.rows[0]);
   return res.rows[0];
 };
 
+const checkNewUser = async (newUser) => {
```

Comment:
rename this function to `doesUsernameExist` and make sure it returns `true` if username exists and `false` otherwise